app/components/views/Home/index.tsx

```diff
@@ -268,7 +274,7 @@ export const Home: FC = () => {
     <nav className={styles.nav}>
       {chainId === 80001 && (
         <p className={styles.testnet_warning}>
-          ⚠️You are connected to <strong>testnet</strong>
+          ⚠️<Trans id="header.connectedto">You are connected to </Trans><strong>testnet</strong>
```

Comment:
This is probably fine for now but it raised a question for me:
Generally speaking, is it better to use interpolation for strings like these?
What does that look like in lingui? in other libs I used its something like:
```json
"header.connectedto": "You are connected to {networkName}"
```

Reply:
the rationale being, in another language the noun `testnet` might not appear at the end

Reply:
If we can't use interpolation in translations I would say this should be a deal-breaker for lingui.

Reply:
Interpolation is definetly better and possible with lingui. I was just being lazy.